src/pipeline_v3/tests/security/test_comprehensive_security.py

```
@@ -0,0 +1,361 @@
+"""
+Comprehensive tests for security module.
+
+Tests all security validators and sanitizers to ensure proper protection
+against common vulnerabilities.
+"""
+
+import ipaddress
```

Ruff (0.11.9)

`ipaddress` imported but unused

Remove unused import: `ipaddress`

(F401)